src/devices/Nmea0183/Sentences/WindSpeedAndAngle.cs

```diff
@@ -0,0 +1,210 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Iot.Device.Common;
+using UnitsNet;
+
+namespace Iot.Device.Nmea0183.Sentences
+{
+    /// <summary>
+    /// MWV sentence: Wind speed and wind angle (true or apparent)
+    /// Note that the wind angle is always given relative to the ship's bow, so to get the wind direction
+    /// in cardinal direction, the heading is required (or with some error, COG can be used)
+    /// </summary>
+    public class WindSpeedAndAngle : NmeaSentence
+    {
+        /// <summary>
+        /// This sentence's id
+        /// </summary>
+        public static SentenceId Id => new SentenceId("MWV");
+        private static bool Matches(SentenceId sentence) => Id == sentence;
+        private static bool Matches(TalkerSentence sentence) => Matches(sentence.Id);
+
+        /// <summary>
+        /// Constructs a new MWV sentence
+        /// </summary>
+        public WindSpeedAndAngle(Angle angle, Speed speed, bool relative)
+            : base(OwnTalkerId, Id, DateTimeOffset.UtcNow)
+        {
+            Speed = speed;
+            Relative = relative;
+            Angle = angle;
+            Valid = true;
+            if (Speed.Unit == UnitsNet.Units.SpeedUnit.Knot)
+            {
+                SpeedUnit = "N";
```

Comment:
I would really use an enum for all those 1 string units. Will make things a bit simpler to use and to read as well as a human